__tests__/app.test.js

```diff
@@ -397,3 +397,24 @@ describe("GET /api/articles (topic query)", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id (Add Feature: comment_count)",()=>{
+  test("GET200: Endpoint responds with selected article by its ID and including it's total count of comment(comment_count)", () => {
+    return request(app)
+      .get("/api/articles/10")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: 10,
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.toBeDateString(),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          comment_count: expect.any(Number)
```

Comment:
Because we've used a flexible jest matcher like toMatchObject here, we don't need to worry about testing for properties that we've already tested for previously, and we shouldn't go back to modify old tests. We could just write a new test testing for the one new property that we have now added and because of the flexible matcher, both tests should pass 👍